timescaledb/quick-start/ruby.md

```diff
@@ -4,727 +4,763 @@ excerpt: Get started with TimescaleDB for a Ruby application
-each item:
+1.  Modify the `home.html.erb` view to iterate over the array and display
+    each item:
 
-```ruby
-<h1>Static Pages requests: <%= PageLoad.count %> </h1>
-```
+     ```ruby
+    <h1>Static Pages requests: <%= PageLoad.count &amp;></h1>
+    ```
 
-Now, each time we refresh our page, we can see that a record is being inserted
-into the `tsdb` TimescaleDB database, and the counter is incremented on the page.
+   Now, each time you refresh the page, you can see that a record is being inserted
+   into the `tsdb` TimescaleDB database, and the counter is incremented on the page.
 
-## Generating requests
+1.  You need to have a lot of page loads to research and explore
+    the [time_bucket] function. You can use [Apache Bench][ab] aka `ab` to
+    request 50,000 times parallelizing 10 times.
 
-We need to have a lot of page loads to continue our research and explore the
-[time_bucket] function.
+    ```bash
+    ab -n 50000 -c 10 http://localhost:3000/static_pages/home
+    ```
 
-Let's use [Apache Bench][ab] aka `ab` to request 50,000 times parallelizing 10
-times.
+   [Apache Bench][ab] creates thousands of records in the hypertable. You can
+   count how many "empty requests" Rails supports.
 
-```bash
-ab -n 50000 -c 10 http://localhost:3000/static_pages/home
-```
+1.  After the `ab` command begins running, you can start a rails console
+    and try some queries using the [time_bucket] function.
 
-Now, you can grab a tea and relax while it creates thousands of records in
-your first hypertable. You'll be able to count how many 'empty requests' your
-Rails supports.
+    ```bash
+    rails console
+    ```
 
-## Counting requests per minute
+1.  View the number of requests per minute:
 
-Once the `ab` command begins running, we can start a rails console
-and try some queries using the [time_bucket] function.
+    ```ruby
+    PageLoad
+      .select("time_bucket('1 minute', created_at) as time, count(1) as total")
+      .group('time').order('time')
+      .map {|result| [result.time, result.total]}
+    ```
 
-```bash
-rails console
-```
+    The result is similar to:
 
-Now, let's start counting how many requests we have per minute:
+    ```ruby
+    PageLoad Load (357.7ms)  SELECT time_bucket('1 minute', created_at) as time, count(1) as total FROM "page_loads" GROUP BY time ORDER BY time
+     =>
+     [2023-02-22 01:32:00 UTC, 6],
+     [2023-02-22 05:57:00 UTC, 3],
+     [2023-02-22 05:59:00 UTC, 75],
+    ```
 
-```ruby
-PageLoad
-  .select("time_bucket('1 minute', created_at) as time, count(1) as total")
-  .group('time').order('time')
-  .map {|result| [result.time, result.total]}
-# => [
-#      [2021-04-14 20:38:00 UTC, 11770],
-#      [2021-04-14 20:39:00 UTC, 11668], ...]
-```
+</Collapsible>
 
-It works! Now, let's create some useful scopes that can help to summarize and
-easily access the `time_bucket` function:
+</Procedure>
 
-## Creating scopes to reuse
+## Create scopes to reuse
```

Comment:
This heading doesn't match the procedure heading, but I'm not sure if you want to go with "Create|Creating scopes to reuse", or "Execute|Executing queries using scopes".